src/main/java/org/codevillage/Neighborhood.java

```diff
@@ -0,0 +1,83 @@
+package org.codevillage;
+
+import java.awt.*;
+
+public class Neighborhood implements Shape
+{
+    private static final Color COLOR = new Color(24, 242, 235, 128);
+    private int centerX;
+    private int centerY;
+    private int width;
+    private int height;
+
+    public Neighborhood()
+    {
+        this.centerX = 0;
+        this.centerY = 0;
+        this.width = 0;
+        this.height = 0;
+    }
+
+    public Neighborhood(int centerX, int centerY, int width, int height)
```

Comment:
Any chance we can have this take in the top-left corner instead of the center? It will make calculating absolute pixel values for positioning easier (currently tracking pixel offsets relative to the top-left for each neighborhood)